src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/CombineSegmentBreakpoints.java

```diff
@@ -14,12 +15,13 @@
 import org.broadinstitute.hellbender.exceptions.UserException;
 import org.broadinstitute.hellbender.tools.copynumber.arguments.CopyNumberStandardArgument;
 import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegion;
+import org.broadinstitute.hellbender.tools.copynumber.utils.annotatedregion.SimpleAnnotatedGenomicRegionCollection;
 import org.broadinstitute.hellbender.utils.IntervalUtils;
 import org.broadinstitute.hellbender.utils.SimpleInterval;
-import org.testng.collections.Lists;
 
 import java.io.File;
 import java.util.*;
+import java.util.function.Consumer;
```

Comment:
Not in this PR, but we removed all short names from CNV tools as part of the kebab-casing effort.  Let's do the same for all tools in this PR.

Reply:
Done.  Also updated `ValidateBasicSomaticShortMutations`